Data/Primitive/ByteArray.hs

```diff
@@ -231,6 +235,39 @@ copyMutableByteArray (MutableByteArray dst#) doff
                     (fromIntegral sz)
 #endif
 
+
+#if __GLASGOW_HASKELL__ >= 708
+-- | Copy a slice of a byte array to an unmanaged address. These must not
+--   overlap. This function is only available when compiling with GHC 7.8
+--   or newer.
+copyByteArrayToAddr
+  :: PrimMonad m
+  => Addr -- ^ destination
+  -> ByteArray -- ^ source array
+  -> Int -- ^ offset into source array
+  -> Int -- ^ number of bytes to copy
+  -> m ()
+{-# INLINE copyByteArrayToAddr #-}
+copyByteArrayToAddr (Addr dst#) (ByteArray src#) soff sz
+  = primitive_ (copyByteArrayToAddr# src# (unI# soff) dst# (unI# sz))
+#endif
+
+#if __GLASGOW_HASKELL__ >= 708
```

Comment:
Why not delete this `#if __GLASGOW_HASKELL__ >= 708` and put `copyMutableByteArrayToAddr` within the same `#if __GLASGOW_HASKELL__ >= 708` block as `copyByteArrayToAddr`?

Reply:
done